### ISSUE TITLE
should we add support for custom losses?

### ISSUE THREAD

**Issue by TobiBu** [robin-janssen/CODES-Benchmark#5]
should we allow custom losses or is this part of the architecture choice?

**Comment by robin-janssen** [comment]
This was - to some degree - addressed in #44. Here, a second loss function ([SmoothL1](https://docs.pytorch.org/docs/stable/generated/torch.nn.SmoothL1Loss.html)) with a configurable parameter beta was added. The choice between standard MSE and SmoothL1 as well as the parameter can be made via the model configuration files. 

In principle, other functions are therefore also supported, but they would have to be added manually as additional choice in a few places.
